demo/vite.config.ts

```diff
@@ -2,7 +2,9 @@ import { defineConfig } from 'vite'
 import * as fs from 'fs'
 import url from 'url'
 
-const cdnDomain = 'http://127.0.0.2:5173'
+//const cdnDomain = 'http://127.0.0.2:5173'
+const cdnDomain = 'http://127.0.0.1:5173' // This fixes the issue with CORS
+
```

Comment:
The whole point is to use another domain for workers

Reply:
Just to run the demo?

Reply:
What do you mean?

Reply:
It may be something wrong i'm doing on my end but when I run the demo with the cdnDomain set to 'http://127.0.0.2:5173' the icons are unable to load due to CORS

Reply:
> 'http://127.0.0.2:5173' the icons are unable to load due to CORS

Do you have some non-default security settings or adblocker? 

With the two separate IPs/domains I do not have an issue with CORS and assets, or workers.

![image](https://github.com/CodinGame/monaco-vscode-api/assets/1828125/58c01887-c9a4-44c4-8147-08a196560fd8)